bot/common/threads/onboarding.py

```diff
@@ -418,27 +419,10 @@ def get_account_verification_prompts(self):
         )
         return [instructions_embed, prompt]
 
-
-class VerifyUserWalletMessageSignatureStep(BaseStep):
```

Comment:
Dropping this class entirely. Verification is now done on save() for the PromptUserWallet step. This is because there were issues in getting the skip emoji to work when having to skip multiple steps. The method used to check for next steps isn't recursive, and only checks the current steps next steps.